talk/morelanguage/morestl.tex

```diff
@@ -25,6 +25,56 @@ \subsection{More STL}
   \end{block}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitlecpp[17]{std::optional}
+  \begin{itemize}
+  \item Indicates the absence of a value with \texttt{std::nullopt}
+  \item Has value semantics (Copy, Move, Compare, stack alloc.)
+  \item Useful in place of pointers where value semantics are intuitive
+  \end{itemize}
+  \begin{exampleblock}{Code example}
+    \begin{cppcode*}{}
+      std::optional<Phone> parse_phone(std::string in)
+      {
+        if (is_valid_phone(in)) {
+          return std::optional<Phone>(in);
+        }
+        return {}; // default constructs std::nullopt
+      }
+      ...
+      auto v = parse_phone(get_input());
+      if (v) {  // alternatively v.is_valid()
+        process_phone(v.value()); // *v is equivalent
+      }
+    \end{cppcode*}
+  \end{exampleblock}
+\end{frame}
+
+\begin{frame}[fragile]
+  \frametitlecpp[17]{std::any}
```

Comment:
I suppose you will fix this slide, which is for the moment a copy of the previous one ?

Reply:
Yes indeed sorry didn't mean to commit that... was planning on adding similar content for std::any and variant